packages/core-js/utils/ast.js

```diff
@@ -0,0 +1,97 @@
+const { findAll } = require('solidity-ast/utils');
```

Comment:
* Moved this file to core-js.
* Only generic AST functions remain.
* Tried to clarify input parameters: when its expecting one ast or many